csvcubed/tests/behaviour/steps/inspectcli.py

```diff
@@ -0,0 +1,117 @@
+from pathlib import Path
+
+from behave import *
+
+from csvcubeddevtools.behaviour.file import get_context_temp_dir_path
+from csvcubed.cli.inspect.metadatainputvalidator import MetadataValidator
+from csvcubed.cli.inspect.metadataprinter import MetadataPrinter
+from csvcubed.cli.inspect.metadataprocessor import MetadataProcessor
+
+
+def _unformat_multiline_string(string: str) -> str:
+    return "".join(s for s in string if ord(s) > 32 and ord(s) < 126)
```

Comment:
I can guess what this is doing - I suspect it's removing colour chars and other non-text stuff that might get printed. It'd be useful to have the exact purpose clarified though.

Reply:
This is now updated.